Week-1/Homework/mandatory/1-writers.js

```diff
@@ -1,3 +1,14 @@
+/* ======= TESTS - DO NOT MODIFY =====
```

Comment:
Could this be extracted to a separate JS file so we don't have to include it at the top of the file where it's quite invasive?